runtime/src/xcm_config.rs

```diff
@@ -202,8 +197,6 @@ impl pallet_xcm::Config for Runtime {
     type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
     type MaxLockers = MaxLockers;
     type MaxRemoteLockConsumers = MaxLockers;
-    #[cfg(feature = "runtime-benchmarks")]
-    type ReachableDest = ReachableDest;
```

Comment:
removal of this was related to fixing the runtime-benchmarks build? Any more info about this change?

Reply:
Exactly. This change was breaking the runtime-benchmarks build